src/docs/cli.md

```diff
@@ -59,7 +67,14 @@ The name of a profile. Can correspond to an entry in credentials.json. A particu
 
 Contains a single object of the format:
 
+```jsx title="Example"
+{
+  "my-first-profile" : {
+    "api_key": "my-api-key"
+  }
+}
 ```
+```jsx title="Schema"
```

Comment:
"jsx" here should be "json" IMO - please make that change a verify everything looks the same and then we gucci